pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -812,18 +807,10 @@ let
         chrome_pgo_phase = 0;
         clang_base_path = "${llvmCcAndBintools}";
       }
-      // (
-        # M134 changed use_qt to use_qt5 (and use_qt6)
-        if chromiumVersionAtLeast "134" then
-          {
-            use_qt5 = false;
-            use_qt6 = false;
-          }
-        else
-          {
-            use_qt = false;
-          }
-      )
+      // {
+        use_qt5 = false;
+        use_qt6 = false;
```

Comment:
Should I leave these in their own block, or should I merge it with the previous one?

Reply:
Merge it with the previous one.